app/_hub/kong-inc/request-transformer/index.md

```diff
@@ -140,6 +140,14 @@ remove --> rename --> replace --> add --> append
 
 ## Examples
 
+<div class="alert alert-info.blue" role="alert">
+  **Kubernetes users:** version `v1beta1` of the Ingress specification does not
```

Comment:
@rainest Same here, change:` **Kubernetes users:** to <strong>Kubernetes users:</strong>`